core/src/queue.rs

```diff
@@ -330,7 +335,7 @@ mod tests {
             let mut domain = Domain::new(domain_id.clone()).build();
             let account_id = AccountId::from_str("alice@wonderland").expect("Valid");
             let mut account = Account::new(account_id, [public_key]).build();
-            account.set_signature_check_condition(SignatureCheckCondition(0_u32.into()));
+            account.set_signature_check_condition(SignatureCheckCondition(false.into()));
```

Comment:
So we didn't notice this, because signature checking didn't work?

Reply:
Yeah. Actually this is the test where I've found signature condition checking bug, because at first it was passing check successfully with `false` as condition